testing/BUILD

```diff
@@ -31,10 +31,10 @@ java_library(
         ],
         "clion-2019.3": glob(["testcompat/v201/com/google/idea/sdkcompat/**/*.java"]),
         "clion-2020.1": glob(["testcompat/v201/com/google/idea/sdkcompat/**/*.java"]),
-        "intellij-2020.1": glob(["testcompat/v201/com/google/idea/sdkcompat/**/*.java"]),
-        "intellij-ue-2020.1": glob(["testcompat/v201/com/google/idea/sdkcompat/**/*.java"]),
```

Comment:
is this deleted intentionally?

Reply:
See upstream issue PR. I don't see any reason to not take it as-is since we don't need to build for 2020.1 from this branch and we won't be cherry-picking or merging these commits for the next release anyway.